packages/jest-core/src/SearchSource.ts

```diff
@@ -111,9 +111,37 @@ export default class SearchSource {
 
     const testCases = Array.from(this._testPathCases); // clone
     if (testPathPattern) {
-      const regex = testPathPatternToRegExp(testPathPattern);
+      let amendedTestPathPattern = testPathPattern;
+
+      const pathSeperator = process.platform === 'win32' ? '\\\\' : path.sep;
+      const relativePathRegex = new RegExp(`[\.]+${pathSeperator}`, 'g');
+      if (relativePathRegex.test(amendedTestPathPattern)) {
+        /**
+         * This handles the scenario where the test path pattern passed is relative
+         * We amend the regular expression to make it optional so that the
+         * test pattern regex will match
+         *
+         * E.g `jest ./hello`
```

Comment:
While I appreciate the comments, to match code style it should be in `// ....` format and not `/** ...`

Reply:
Sure thing.